setup/setup-ci/security-scanner/README.md

```diff
@@ -0,0 +1,181 @@
+---
+page_type: sample
+languages:
+- bash
+- python
+products:
+- azure-machine-learning
+description: Sample setup script to scan Compute Instances for malware and security vulnerabilities
+---
+
+# Compute Instance Security Scanner
+
+[![license: MIT](https://img.shields.io/badge/License-MIT-purple.svg)](../../../LICENSE)
+
+A security scanner for Azure ML [Compute Instances](https://learn.microsoft.com/en-us/azure/machine-learning/concept-compute-instance) reporting malware and vulnerabilities in OS and Python packages to [Azure Log Analytics](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-overview). For details on the vulnerability management process for the Azure Machine Learning service, see [Vulnerability Management](https://learn.microsoft.com/azure/machine-learning/concept-vulnerability-management).
```

Comment:
Details on how this scanner works are important. The assessments section isn't even its own section heading, it is a footnote in the Telemetry section.

What this scanner does to your CI and how should be up front and center. The Assessments section should be up here, before we even get started. It should let you know that you'll be installing ClamAV and Trivy, as well as the configurations of those packages (for example, ClamAV can be configured for real time scanning. If it is, then that has performance implications for the CI.)

Reply:
Added an assessment section detailing what is installed and run on the machine.